frontend/src/utils/layoutManager.ts

```diff
@@ -36,19 +33,15 @@ class LayoutManager {
       smallMaxHeight: Infinity,
       bigMaxWidth: Infinity,
       bigMaxHeight: Infinity,
-      bigMaxRatio: shouldMakeLargeTilesLandscape ? 9 / 16 : 3 / 2,
+      bigMaxRatio: 9 / 16,
```

Comment:
I'm not sure if this is what we want for screenshare, can you please check? In initial tests of sharing a thing app window, I see a regression where the screenshare is smaller than before. 
If you want to lock in video subs at this ratio then I'd suggest keeping the parameter from before but removing the pinning logic and replacing it with just a screenshare check